config/navigation.txt

```diff
@@ -60,19 +60,22 @@ Learn
     [/learn/deploy/deploy-with-balena-button]
 
   MANAGE
-    [/learn/manage/account]
-    [/learn/manage/organizations]
     [/learn/manage/apps]
```

Comment:
Maybe not something we need to do in this PR, but I think maybe we should move the "Apps" section lower in the Manage section, since it's not currently super applicable to our current balenaCloud user usecase and having it as the most prominent first item here give it a lot of imporatance.

Reply:
Resolved that